lib/lolcommits/platform.rb

```diff
@@ -1,68 +1,63 @@
 # -*- encoding : utf-8 -*-
 require 'mini_magick'
+require 'rbconfig'
 
 module Lolcommits
   class Platform
     # A convenience name for the platform.  Mainly used for metaprogramming.
     # @return String
     def self.platform
-      if platform_fakeplatform?   then ENV['LOLCOMMITS_FAKEPLATFORM']
-      elsif platform_fakecapture? then 'Fake'
-      elsif platform_mac?         then 'Mac'
-      elsif platform_linux?       then 'Linux'
-      elsif platform_windows?     then 'Windows'
-      elsif platform_cygwin?      then 'Cygwin'
+      if platform_mac?        then 'Mac'
+      elsif platform_linux?   then 'Linux'
+      elsif platform_windows? then 'Windows'
+      elsif platform_cygwin?  then 'Cygwin'
       else
         fail 'Unknown / Unsupported Platform.'
       end
     end
 
+    # The capturer class constant to use
+    # @return Class
+    def self.capturer_class(animate = false)
+      Object.const_get('Lolcommits').const_get(
+        ENV['LOLCOMMITS_CAPTURER'] || "Capture#{platform}#{animate ? 'Animated' : nil}"
```

Comment:
I wonder if we are getting enough benefit from the metaprogramming magic we have historically used here (in the second half of the statement) versus just a more traditional switch/case logic since we don't have too many possible options.  If we did the later we could get rid of the entire `self.platform` method I believe, and we would be less reliant on strings to communicate options internally?  Just a thought, I know we've done it this way for a while and it hasn't caused any problems.